petab/petab_schema.yaml

```diff
@@ -0,0 +1,46 @@
+# For syntax see: https://json-schema.org/understanding-json-schema/index.html
+#$schema: "https://json-schema.org/draft/2019-09/meta/core"
+$schema: "http://json-schema.org/draft-06/schema"
+description: PEtab parameter estimation problem config file schema
+
+properties:
+
+  petab_version:
+    type: string
+    description: Version of the PEtab format (e.g. 0.0.0a16)
+
+  parameter_file:
+    type: string
+    description: |
+      PEtab parameter file name containing parameters of all models listed in
+      `problems`. Absolute or relative path.
+
+  problems:
+    type: array
+    description: |
+      One or more (SBML model, measurement, condition files) tuples.
+    items:
+      type: object
+      description: PEtab SBML, condition and measurement tables
+      properties:
+        sbml_file:
+          type: string
+          description: PEtab SBML file
+        measurement_files:
+          type: array
+          description: List of PEtab measurement files
+          items:
+            type: string
+            description: PEtab measurement file
```

Comment:
analogous to measurement file, also specify the (optional) simulation file here? depends on whether we want to regard these as parts of the petab problem.

Reply:
"simulation file" the place to write results to? I think I'd keep that out of here.

Reply:
yes, I man the file that has the same format as measurement_file and contains the simulations. I think that exists now for various benchmark models, would be the question if we want to have it as optional for the standard now. point in favor would be that this can further automatize figure generation.

Reply:
Fair point. However, this may need some more thoughts. In case of multi-start optimization we may want to have multiple result files. The we need to allow for some kind of wildcards. This is again quite tool specific. Would tackle that later.

For the benchmark collection, where exactly one file is present, I'd handle that with a separate file with benchmark-specific info.

More important with respect to the visualization would be providing the visualization schema. Will leave that to people working on viz.